wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -1,3 +1,12 @@
 <body>
-    Welcome to IIIT Vadodara.
-</body>
\ No newline at end of file
+  Welcome to IIIT Vadodara.
+  {% if latest_announcement %}
+  <ul>
+    {% for announcement in latest_announcement %}
+    <li><a href='/announcement/{{ announcement.key }}'> {{ announcement.title }}</a></li>
+    {% endfor %}
+  </ul>
+  {% else %}
+  <p> No latest announcement </p>
+  {% endif %}
+</body>
```

Comment:
Oh, yes. Sorry, I referenced wrong file. Pagination is required at `/announcement/`.